src/Explorer/Panes/SettingsPane/SettingsPane.tsx

```diff
@@ -41,12 +41,18 @@ export const SettingsPane: FunctionComponent = () => {
       ? LocalStorageUtility.getEntryNumber(StorageKey.MaxDegreeOfParellism)
       : Constants.Queries.DefaultMaxDegreeOfParallelism
   );
+  const [priorityLevel, setPriorityLevel] = useState<string>(
+    LocalStorageUtility.hasItem(StorageKey.PriorityLevel)
+      ? LocalStorageUtility.getEntryString(StorageKey.PriorityLevel)
+      : Constants.PriorityLevel.Low
```

Comment:
Add a constant for the Data Explorer's default priority level and use that everywhere

Reply:
Changes have been committed where we have replaced the None with the Default